front/lib/plans/subscription.ts

```diff
@@ -393,6 +408,33 @@ export const getCheckoutUrlForUpgrade = async (
   };
 };
 
+export async function isStripeSubscriptionOnProPlan(
+  stripeSubscription: Stripe.Subscription
+): Promise<boolean> {
+  const { data: subscriptionItems } = stripeSubscription.items;
+  const proPlanStripeProductId = getProPlanStripeProductId();
+
+  return subscriptionItems.some(
+    (item) => item.plan.product === proPlanStripeProductId
+  );
+}
+
+export async function isSubscriptionOnProPlan(
```

Comment:
Do we need to export?

Reply:
I'll be using it on another file on my next PR!